### DISCUSSION TITLE
Some questions from a new comer

### DISCUSSION THREAD

**Discussion by wiiznokes** [helix-editor/helix#10262] [Q&A]
I started using Helix today and i love it so far, except some things i find weird/frustrating:
- why cutting + pasting some text doesn't restore the initial state ?
    i.e, if i have`helix.` , and i cut just `helix`, then espace + paste, i will have `.helix`.
- why cutting active the INSERT mode ?
- why cutting + copying + pasting doesn't ALSO use the system clipboard by default (register and clipboard combined)?
- why pasting some text create a new line??? (i think it is because of the ending \n. But the cursor bar in NORMAL mode doesn't work, so this make super difficult for me to see things

**Comment by gabydd** [comment]
For 1 use ``P`` which will paste before the selection
2. c is change not cut you probably want ``d`` which will yank and delete

**Comment by wiiznokes** [comment]
Thank! It could be simpler if the cursor was a bar but that resolver my issue
